models/notification.go

```diff
@@ -124,6 +126,18 @@ func (message *Notification) checkConfig() {
 	message.Recipients = newRecipient
 }
 
+func cleanNotificationDescription(content string) string {
+	newContent = strings.ReplaceAll(content, "\n", "")
+	newContent = regexp.MustCompile(`#{1,2}\d+`).ReplaceAllString(newContent, "")
+	newContent = regexp.MustCompile(`\${1,2}.*?\${1,2}`).ReplaceAllString(newContent, "[公式]")
+	newContent = regexp.MustCompile(`!\[\]\(dx_\S+\)`).ReplaceAllString(newContent, "[表情]")
+	newContent = regexp.MustCompile(`!\[.*?\]\(.*?\)`).ReplaceAllString(newContent, "[图片]")
```

Comment:
`MustCompile` is often used for global variable initialization. Have you considered made the compiled regex static for better performance?